default/AI/charting/charts.py

```diff
@@ -20,7 +20,7 @@
 from glob import glob
 import traceback
 
-dataDir = os.environ.get('HOME', "") + "/.freeorion"
+dataDir = (os.environ.get('HOME', "") + "/.freeorion" ) if os.name != 'posix' else (os.environ.get('XDG_DATA_HOME', os.environ.get('HOME', "") + "/.local/share") + "/freeorion") 
```

Comment:
That reason is just fine with me.  I just asked because I don't have a  complete overview over the whole code base and that part of the part the PR looks a bit weird without looking too deep into the context.